src/plugins/data/public/query/lib/from_user.ts

```diff
@@ -28,6 +28,10 @@ import _ from 'lodash';
 export function fromUser(userInput: object | string) {
   const matchAll = '';
 
+  if (_.isEmpty(userInput)) {
+    return '';
+  }
```

Comment:
Was this just an edge case that surfaced from converting the unit tests to TS, or is there another reason for adding this check?

Reply:
Okay, just leaving a follow-up note here for posterity, based on our offline conversation:

After we tested this, we discovered that converting the unit test for this function from mocha to jest caused one of the tests to fail, presumably due to differences between the jest & expect assertions.

So this is actually fixing a bug that was previously uncaught by the unit tests, and is the way the function was intended to work originally.